docs/integrations/saas-cloud/confluent-cloud.md

```diff
@@ -0,0 +1,119 @@
+
+## Metrics type  
+
+The Confluent Cloud app uses metrics:
+* [Confluent Cloud runtime metrics](https://api.telemetry.confluent.cloud/docs/descriptors/datasets/cloud).
+
+### Collect Metrics
+
+Sumo Logic supports collecting metrics by C2C source:
+
+* Configure a [Confluent Cloud Metrics Source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/confluent-cloud-metrics-source/)
+
+## Installing the Confluent Cloud  
+
+Now that you have set up a collection for **Confluent Cloud**, install the Sumo Logic app to use the pre-configured [dashboards](#viewing-the-confluent-cloud-dashboards) that provide visibility into your environment for real-time analysis of overall usage.
+
+import AppInstall from '../../reuse/apps/app-install-v2.md';
+
+<AppInstall/>
+
+## Viewing the Confluent Cloud dashboards  
+
+### Confluent Cloud - Compute Pool
+
+Confluent Cloud - Compute Pool dashboard provides details on the status and utilization of compute resources in your Confluent Cloud environment. Use this dashboard to:
+
+* Monitor the average Flink statement status to identify potential issues with stream processing jobs.
+* Track CPU utilization against the set limit to ensure optimal resource allocation and prevent performance bottlenecks.
+* Analyze trends in CPU minutes consumed over time to optimize cost and capacity planning for your Confluent Cloud deployment.
+* Compare current CPU usage across different compute pools to balance workloads and improve overall system efficiency.
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/ConfluentCloud/Confluent-Cloud-Compute-Pool.png')} alt="Confluent Cloud - Compute Pool" style={{border: '1px solid gray'}} width="800" />
+
+### Confluent Cloud - Connectors
+
+Confluent Cloud - Connectors dashboard provides details on the status, performance, and health of Confluent Cloud connectors.
+
+Use this dashboard to:
+* Monitor the status of connectors and their tasks to quickly identify any that are failed, paused, or provisioning.
+* Track the volume of sent and received records across different connectors to ensure data flow is as expected.
+* Analyze the average sent and received bytes to detect any unusual spikes or drops in data transfer.
+* Identify potential bottlenecks by correlating connector status with sent/received records and bytes.
+* Monitor dead letter queue records to catch any messages that failed to be processed correctly.
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/ConfluentCloud/Confluent-Cloud-Connectors.png')} alt="Confluent Cloud - Connectors" style={{border: '1px solid gray'}} width="800" />
+
+### Confluent Cloud - Kafka Cluster
+
+Confluent Cloud - Kafka Cluster dashboard provides details on key performance metrics and operational statistics for your Kafka cluster in Confluent Cloud.
```

Comment:
apply the same change as https://github.com/SumoLogic/sumologic-documentation/pull/5495/files#r2172504296